config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+
+  get "/movies/:id", to: "movies#show"
+  get "/movies/:id/movie_actors/new", to: "movie_actors#new"
```

Comment:
This /new path is actually not necessary because we're rendering the form on the show page. We only need a new page if we're rendering the form to create a new resource on a separate page